java/src/test/java/ai/onnxruntime/InferenceTest.java

```diff
@@ -666,6 +672,14 @@ public void testOpenVINO() throws OrtException {
 
   @Test
   @EnabledIfSystemProperty(named = "USE_DNNL", matches = "1")
+  // TODO see if this can be enabled on Windows.
+  // Error in CI build:
+  // ai.onnxruntime.OrtException: Error code - ORT_RUNTIME_EXCEPTION - message:
+  // D:\a\_work\1\s\onnxruntime\core\session\provider_bridge_ort.cc:1530
+  // onnxruntime::ProviderLibrary::Get [ONNXRuntimeError] : 1 : FAIL : LoadLibrary failed with error
+  // 126 "" when trying to load
+  // "C:\Users\cloudtest\AppData\Local\Temp\onnxruntime-java9085185608411256214\onnxruntime_providers_dnnl.dll"
+  @DisabledOnOs(value = OS.WINDOWS)
```

Comment:
note: this is currently disabled on Windows without this change because the system properties aren't coming through. it would be good to get it working though.

Reply:
I can try and build it locally on Windows to figure out the dependency that needs to be added. Does the Python DNNL test have a different library search path?

Reply:
sorry, I'm not familiar with the DNNL setup. let me see if I can find someone on the team who knows more about it.

Reply:
@jywu-msft any idea about this?

Reply:
i don't know the context here. how is DNNL related to enabling QNN on Java.
error seems to indicate it can't load DNNL EP shared lib (either that lib isn't found or its dependencies - presumably dnnl libs.  which should be built from source when --use_dnnl is set in build options)

Reply:
The DNNL enable argument wasn't being passed through the Java build properly, @edgchen1 fixed the build process so things are passed through more cleanly and now this test fails on Windows as it's being triggered correctly. So it's been disabled, and it would be nice to re-enable it, but to do that we need to figure out which dependency it is missing.

Reply:
FYI, I tried a DNNL build locally and can reproduce the issue. the onnxruntime_providers_dnnl.dll is indeed not present in that temp directory in the error message. the temp directory does contain onnxruntime.dll and onnxruntime4j_jni.dll.

the normal build output directory (e.g., `build/Debug/Debug` on Windows) contains onnxruntime_providers_dnnl.dll and dnnl.dll, in addition to the ones in the temp directory above.

I'm not sure how that temp directory gets generated.

I'll probably leave this test disabled for this PR.